src/test/java/bio/terra/service/filedata/google/firestore/FileLoadTest.java

```diff
@@ -135,6 +137,43 @@ public void fileLoadTest() throws Exception {
 
         assertThat(summary.getSucceededFiles(), equalTo(filesToLoad));
     }
+
+    // The purpose of this test is to have a long-running workload that completes successfully
+    // while we delete pods and have them recover.
```

Comment:
Since I need to start writing more tests myself, I just wanted to learn more about this.
Are you actually able to test deleting pods & recovering them? I didn't see any associated code that did that, so just wondering how that worked!

Reply:
It is possible to do that, but I haven't written any test code to do it. There's probably some test infrastructure work to do to pass the target namespace into the test.

For now, this is meant to be a background workload while I manually fuss with the pods.

Reply:
Gotcha, makes sense! Thanks!